src/d2lfetch-simple-cache.js

```diff
@@ -110,4 +125,18 @@ export class D2LFetchSimpleCache {
 				return response;
 			});
 	}
+
+	_getKey(request) {
+		const key = request.method + request.url;
+		if (request.headers.has('Authorization')) {
+			return key + request.headers.get('Authorization');
+		}
+
+		return key;
+	}
+
+	_reset() {
```

Comment:
Added this so the tests can reset in between tests and don't interfere with each other.